client/my-sites/plans/main.jsx

```diff
@@ -36,9 +41,54 @@ const Plans = React.createClass( {
 		};
 	},
 
+	shouldDisplayJetpackPlansDisabled() {
+		const { selectedSite } = this.props;
+
+		return selectedSite && selectedSite.jetpack && ! isEnabled( 'manage/jetpack-plans' );
+	},
+
+	componentDidMount() {
+		const { context, selectedSite } = this.props,
+			analyticsPageTitle = 'Plans',
+			basePath = route.sectionify( context.path ),
+			analyticsBasePath = ( selectedSite ) ? basePath + '/:site' : basePath;
+
+		// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
+		context.store.dispatch( setTitle( i18n.translate( 'Plans', { textOnly: true } ) ) );
```

Comment:
Any reason why we're dispatching this from context? We can pass a second argument to connect, [mapDispatchToProps](https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-optionshttps://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options) that allows us to avoid this.

eg in connect
```
export default connect( ( state ) => {}, { setTitle } )( localize( Plans ) );
```

and here:
```
this.props.setTitle( translate( 'Plans' ) );
```